### PR TITLE
Added WIP script to update accounts for bootstrapping

### PR DESCRIPTION
**Pull request by andrewpatto**

### CODE REVIEW

**Comment by andrewpatto** [comment]
Another thing we could do is have some sort of "master" data structure (JSON?) naming a labelling every account.

And from the data structure we could do things like generate the appropriate aws config dynamically (would be a bit of jq).

So at the moment I think here we would be committing to keeping this master config ini file (and william has one very similar for his scripts)

**Comment by andrewpatto** [comment]
But anyhow this is just a starting point

File: scripts/all-accounts-aws-config.ini [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+sso_session = umccr-bootstrap-session
+sso_account_id = 843407916570
+sso_role_name = AdministratorAccess
+region = ap-southeast-2
+
+[profile unimelb-australiangenomics]
+sso_session = unimelb-bootstrap-session
+sso_account_id = 258177526432
+sso_role_name = AWSAdministratorAccess
+region = ap-southeast-2
```
Comment (andrewpatto) [inline scripts/all-accounts-aws-config.ini:33]: So if we are going to do this kind of script (also see the security scripts) - we probably should have some common master definition of all of this

Comment (victorskl) [reply scripts/all-accounts-aws-config.ini:33]: Yep. Agree, thought of the same.

**Review by victorskl — APPROVED** [review]
Thanks for starter, Patto. Looks good from my side. 

I hope, you could kick start in new AG account there, first, pls. We will catchup in tick after Ocicat.

**Comment by andrewpatto** [comment]
Yes AG has been bootstrapped

**Comment by victorskl** [comment]
[UoM AWS Migration: Bootstrap CDK in new UoM AWS accounts](https://trello.com/c/jn56wL6f/1439-uom-aws-migration-bootstrap-cdk-in-new-uom-aws-accounts)

File: scripts/all-accounts-aws-config.ini (hunk 1 above)
Comment (andrewpatto) [reply scripts/all-accounts-aws-config.ini:33]: A master JSON that we can JQ into various forms??

Comment (victorskl) [reply scripts/all-accounts-aws-config.ini:33]: Sure

**Comment by reisingerf** [comment]
Awesome, thanks!
I agree, there's lots of potential to build on. I think sharing the same config is a good first step that also promotes adoption of the same naming convention.
I think we may also be able to list all accounts we have access to using the SSO, so could dynamically discover and bootstrap them...
Account management and initial setup is also a topic we have brought up with UoM, but never resolved. Perhaps they'd be happy to do the bootstrapping for us (I'm probably dreaming here).

**Review by reisingerf — APPROVED** [review]

**Pull request merged**